soc/nordic/common/uicr/gen_uicr.py

```diff
@@ -440,6 +440,36 @@ def main() -> None:
         type=int,
         help="Protected memory size in bytes (must be divisible by 4096)",
     )
+    parser.add_argument(
+        "--wdtstart",
+        action="store_true",
+        help="Enable watchdog timer start in UICR",
+    )
+    parser.add_argument(
+        "--wdtstart-instance-code",
+        type=lambda s: int(s, 0),
+        help="Watchdog timer instance code (0xBD2328A8 for WDT0, 0x1730C77F for WDT1)",
+    )
+    parser.add_argument(
```

Comment:
Should we use a sub-parer for the fields that are only relevant if ``--wdtstart`` is enabled?

I see that we handle this in kconfig, but the script would be more robust for isolated use with that change IMO.